src/main/python/janusgraph_python/serializer/RelationIdentifierSerializer.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONUtil
+
+
+class RelationIdentifierSerializer(object):
+    """
+    This is serializer method being used to serialize RelationIdentifier object which is used by JanusGraph for edge ID
+    """
+
+    GRAPHSON_PREFIX = "janusgraph"
+    GRAPHSON_BASE_TYPE = "RelationIdentifier"
+
+    @classmethod
+    def dictify(cls, relationID, writer):
+        """ Serializes RelationIdentifier object.
+
+        Args:
+            relationID (RelationIdentifier): The RelationID to serialize.
+            writer:
+
+        Returns:
+            json
```

Comment:
Actually, commenting helps me, because I work on lot of things at same time. If I touch upon this after few months, without comments I'll be lost @FlorianHockmann . Plus helps new guy to write their own serializer and deserializer if you have comments. But as you said its coding habit :-)

Prob with lot of comments is it needs lot of reviews, like you must have realized by now already the spelling mistakes and JanusGraph Server's various forma :-P

Reply:
I'm not against comments in general. I was just referring to these API comments that are often just copied around and therefore provide not much value when reading the code in my opinion.

Yeah, I definitely commented on a lot of small nits here that result in another review round and make all of this quite time consuming. Maybe it would help if you would just review the PR again yourself? Then you can also directly check whether it confirms to PEP 8.
I personally made it a habit to review a diff completely on GitHub before I actually create a PR as I often find some issues that I didn't notice when directly working on the code.